csi/node_server.go

```diff
@@ -456,6 +456,26 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		devicePath = cryptoDevice
 	}
 
+	if volumeCapability.GetBlock() != nil {
+		if err := ns.nodeStageBlockVolume(volumeID, devicePath, targetPath, mounter); err != nil {
```

Comment:
NIT: Just found that the staging path is named `targetPath` in this function. I am thinking if it's better to rename it

Reply:
Rename to `stagingTargetPath` instead. Thank you.